drivers/sensor/bme680/Kconfig

```diff
@@ -0,0 +1,157 @@
+# Kconfig.bme680 - BME680 temperature, pressure, humidity and gas sensor configuration options
+
+#
+# Copyright (c) 2016, 2017 Intel Corporation
+# Copyright (c) 2017 IpTronix S.r.l.
+# Copyright (c) 2018 Bosch Sensortec GmbH
```

Comment:
it's a new file with newly created options in here, so put only your copyright

Reply:
Hi @tbursztyka,

The BME680 driver was written by taking several snippets and the overall structure from the BME280 driver.  Hence the copyright for the previous developers was maintained. Can be removed if not necessary.